### PR TITLE
Project Submission

### PR DESCRIPTION
**Pull request by silviatofana**
https://www.loom.com/share/88fab7327a264e2a8525db1e0871020e
Design

You should follow these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), including:
Colors.
Typography: font face, size and weight.
Layout: composition and space between elements.
Interactions

Splash screen

A simple page with the name of your app (yes, you need to choose one), and links to the sign up and log in pages.
Sign up and log in pages

The user should be able to register in the app with full name, email and password (all mandatory).
The user can log into the app using email and password.
If the user is not logged in, they can't access pages that require the user to be logged in (all the pages described below).
Home page (catagories page)

When the user logs in, they are presented with the categories page.
For each category, the user can see their name, icon and the total amount of all the transactions that belongs to that category.
When the user clicks (or taps) on a category item, the application navigates to the transactions page for that category.
There is a button "add a new category" at the bottom that brings the user to the page to create a new category.
Transactions page

For a given category, the list of transactions is presented, ordered by the most recent.
At the top of the page the user could see the total amount for the category (sum of all of the amounts of the transactions in that category).
There is a button "add a new transaction" at the bottom that brings the user to the page to create a new transaction.
When the user clicks on the "Back" button (<), the user navigates to the home page.
Testing requirements

Create unit and integration tests for all the most important components of your RoR application.
Technical requirements

You should use Postgres as your database.
You should use devise for authentication.
You should validate all user input to make sure that anyone with bad intentions cannot compromise your app.
You can use a view template engine of your choice (.erb, .slim, .haml).
The project should be deployed and accessible online.
Your database schema should reflect the following structure:

image

### CODE REVIEW

File: app/views/devise/sessions/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```
Comment (DammyShittu) [inline app/views/devise/sessions/new.html.erb:26]: - [ ] Kindly verify that you deployed your app. Your local app is radically different from the deployed app and your local version does not have a splash screen. Kindly inspect that.

  Local version login page | Deployed version login page
  :---:|:---:
  klo | lof

File: config/routes.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  get 'entities/create'
+  get 'entities/update'
+  get 'entities/destroy'
+  get 'entities/edit'
+  devise_for :users
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  # root "articles#index"
+  root to: 'users#index'
```
Comment (DammyShittu) [inline config/routes.rb:14]: - [ ] Your root path seems to be different for the deployed and local versions. From the requirements your root path should point to the categories list whereas your local version points to the users index. Kindly ensure your app is working properly

  local version root route | Deployed version root path
  :---:|:---:
  fhju | dmk

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @silviatofana,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

In my opinion, your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by silviatofana** [comment]
thankssssssssssssssssssssssssssssss

**Pull request merged**